libs/category/src/filters/validators/request-name-match.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { TestBed } from '@angular/core/testing';
+
+import {
+  DaffCategoryFilterRequest,
+  DaffCategoryFilterNotFound,
+  DaffCategoryFilter,
+} from '@daffodil/category';
+import {
+  DaffCategoryFilterFactory,
+  DaffCategoryFilterRequestFactory,
+} from '@daffodil/category/testing';
+
+import { daffCategoryValidateFilterRequestNameMatch } from './request-name-match';
+
+describe('@daffodil/category | filters | validators | daffCategoryValidateFilterRequestNameMatch', () => {
+  let filterFactory: DaffCategoryFilterFactory;
+  let filterRequestFactory: DaffCategoryFilterRequestFactory;
+
+  let filterRequest: DaffCategoryFilterRequest;
+  let filter: DaffCategoryFilter;
+
+  beforeEach(() => {
+    filterFactory = TestBed.inject(DaffCategoryFilterFactory);
+    filterRequestFactory = TestBed.inject(DaffCategoryFilterRequestFactory);
+
+    filterRequest = filterRequestFactory.create();
+  });
+
+  describe('when the filter is not present', () => {
```

Comment:
These tests can be a little more specific to include the message of the error

Reply:
I think @damienwebdev specifically did not want to assert the error message.

Reply:
Correct, modifying the error message isn't particularly worthy of a test failure.

Reply:
Then why do we have two different error messages? If changing an error message isn't worthy of a test failure, then we shouldn't need two different messages

Reply:
That's a good point -- @griest024 when would `!filter` ever be true?